product-features/token-swaps/how-to.md

```diff
@@ -0,0 +1,134 @@
+
+#### Transaction Settings
+
+If you want to adjust slippage tolerance, select the "Settings" icon to open the Transaction Settings panel. Set your desired tolerance and click "Confirm". This will determine your allowed range for price movement. Your transaction will revert if the price changes unfavourably by more than this percentage.&#x20;
+
+<figure><img src="../../.gitbook/assets/token-swaps/3-tx-settings-icon.png" alt="" width="375"><figcaption><p>Transaction Settings icon.</p></figcaption></figure>
+
+<figure><img src="../../.gitbook/assets/token-swaps/3-tx-settings.png" alt="" width="375"><figcaption><p>Transaction Settings panel example, with slippage tolerance set to 2%.</p></figcaption></figure>
+
+### Step 4
+
+Once you're ready to move ahead, select the "Swap" button which will bring up the Confirmation panel. This panel provides a final overview of your transaction details, allowing you to double-check price, route, fees and slippage. If everything looks good, click "Confirm" 😎.
+
+<figure><img src="../../.gitbook/assets/token-swaps/4-confirmation-panel.png" alt="" width="375"><figcaption></figcaption></figure>
+
+### Step 5
+
+After clicking "Confirm", you will need to confirm the transaction in your wallet. Here, your Stacks wallet is interacting with ALEX smart contract and is asking you for approval. Scroll through the wallet transaction window, review it and confirm the transaction. By doing this, you are allowing the wallet to sign and broadcast the transaction.
+
+{% hint style="info" %}
+To be completely sure, you can check:
+
+* The transaction is requested by **"Alex app" (app.alexlab.co)**
+* The transfer amounts, covered by [Stacks post conditions](https://docs.stacks.co/stacks-101/post-conditions). If these conditions are not met, the transaction will abort. Note:
+  * The amount you transfer to the smart contract is exactly determined (STX in the example).
+  * The amount the smart contract transfers to you (ALEX in the example) is subject to an "equal to or greater than" condition. This accounts the potential slippage variation, and here you can see the exact lower bound.
+{% endhint %}
+
+<div>
+
+<figure><img src="../../.gitbook/assets/token-swaps/5-swap-helper-post-con.png" alt="" width="375"><figcaption><p>Transfer amounts involved and post conditions.</p></figcaption></figure>
+
+ 
+
+<figure><img src="../../.gitbook/assets/token-swaps/5-swap-helper-function-args.png" alt="" width="375"><figcaption><p>Function arguments and confirmation button.</p></figcaption></figure>
+
+</div>
+
+### Step 6 <a href="#step-7" id="step-7"></a>
+
+Wait for the transaction to be confirmed on the network.
+
+{% hint style="info" %}
+Recommended to track transaction status:
+
+* Turn on Telegram notifications, you will get notified when the transaction is confirmed.
+* Search for the transaction on [explorer](https://explorer.hiro.so).
+* Check your address activity on the wallet.
```

Comment:
https://app.alexlab.co/explorer

we have recently added Explorer as a new feature, where all confirmed(including failed) ALEX-related txs will be displayed on the explorer. Users are encouraged to use them as well

Reply:
Nice. Changed!